daras_ai_v2/slack_bot.py

```
@@ -685,3 +685,69 @@ class SlackAPIError(Exception):
+    original_blocks = original_blocks or []
+    new_blocks = [b for b in original_blocks if b.get("type") != "actions"]
+    if selection_text:
+        new_blocks.append(
+            {
+                "type": "context",
+                "elements": [
+                    {"type": "mrkdwn", "text": f"_You chose: {selection_text}_"},
+                ],
+            }
+        )
+    res = requests.post(
```

Ruff (0.14.1)

Probable use of `requests` call without timeout

(S113)